mirrormanager2/app.py

```diff
@@ -373,14 +375,22 @@ def host_new(site_id):
 
         try:
             SESSION.flush()
+            status = 'Host added'
```

Comment:
I fear this might be a little confusing, the topic is ``mirror.added`` and the status says ``Host added``, maybe we can just change the topic to represent the action ``host.added``, what do you think?